schemas/messaging-system/cfr/cfr.1.schema.json

```diff
@@ -0,0 +1,69 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "anyOf": [
+    {
+      "required": [
+        "message_id"
+      ]
+    },
+    {
+      "required": [
+        "bucket_id"
+      ]
+    }
+  ],
+  "properties": {
+    "addon_version": {
+      "type": "string"
+    },
+    "bucket_id": {
+      "type": "string"
+    },
+    "client_id": {
+      "pattern": "^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}$",
+      "type": "string"
+    },
+    "event": {
+      "description": "An event identifier",
+      "type": "string"
+    },
+    "impression_id": {
+      "description": "An alternative UUID used when client_id is absent",
+      "pattern": "^\\{[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}\\}$",
+      "type": "string"
+    },
+    "locale": {
+      "type": "string"
+    },
+    "message_id": {
+      "type": "string"
+    },
+    "profile_creation_date": {
+      "type": "integer"
+    },
+    "release_channel": {
+      "type": "string"
+    },
+    "shield_id": {
+      "description": "A semicolon separated string to store a list of Shield study IDs",
+      "type": "string"
+    },
+    "value": {
```

Comment:
Can we use a better name like `event_context`?

Reply:
Yep, we can do that.